src/components/DAO/Tokenomics.tsx

```diff
@@ -0,0 +1,727 @@
+                  </svg>
+                </a>
+              </div>
+            </div>
+          </div>
+
+          <div className="grid md:grid-cols-2 lg:w-2/3 gap-2 w-full">
+            <div className="w-full h-full bg-[#2c2f38] rounded-md p-3 relative shadow-sm border border-gray-700">
+              <table className="text-sm h-[215px] text-[#8D8E96]">
+                <tbody>
+                  <tr>
+                    <td className="min-w-[95px]">Name: </td>
+                    <td>{getTokenData(SDIV[0])?.name} </td>
+                  </tr>
+                  <tr>
+                    <td>Symbol: </td>
+                    <td>{getTokenData(SDIV[0])?.symbol} </td>
+                  </tr>
+                  <tr>
+                    <td>Address: </td>
+                    <td className="overflow-clip">
+                      <ShortAddress address={getTokenData(SDIV[0])?.address} />
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>Total supply: </td>
+                    <td>{poolData.sdivTotalSupply} SDIV</td>
+                  </tr>
+                  <tr>
+                    <td>Wallet: </td>
+                    <td>{poolData.sdivBalance} SDIV</td>
+                  </tr>
+                  <tr>
+                    <td>Earned: </td>
+                    <td>{poolData.sdivEarned} SDIV</td>
+                  </tr>
+                </tbody>
+              </table>
+
+              <img
+                className="rounded-full absolute right-3 top-3 w-1/6 lg:w-1/5"
+                src={getTokenData(SDIV[0])?.logoURI}
+                alt={getTokenData(SDIV[0])?.logoURI}
+                title={getTokenData(SDIV[0])?.symbol}
+              />
+
+              <div className="flex mt-3 text-sm">
+                {" "}
+                <button
+                  onClick={() => {
```

Comment:
onClick={harvest}

Reply:
if i dont add ()=> it automatically calls at first render